bower.json

```diff
@@ -17,9 +17,24 @@
     "package.json"
   ],
   "dependencies": {
```

Comment:
Can you confirm that all of these dependencies are the latest versions, and that they have no conflicts?